src/runtime/types.ts

```diff
@@ -86,6 +86,16 @@ export interface ModuleOptions {
     | 'bottom-left'
     | 'bottom-right'
     | 'bottom-full'
+  /**
+   * Position of the control button.
+   * 'top-left', 'top-right', 'bottom-left', 'bottom-right', or undefined.
```

Comment:
```suggestion
   * 'top-left', 'top-right', 'bottom-left' or 'bottom-right'.
```
Would that fit better?